.github/workflows/forge-pfn.yaml

```diff
@@ -95,49 +95,10 @@ jobs:
 
   ### Public fullnode tests
 
-  # Measures PFN latencies with a constant TPS
```

Comment:
Anyone reason we're removing these? One of the things I liked about them was that it was easy for us see/debug changes in performance/latency across tests (i.e., because we only change 1 variable at a time). Also, given that we're no longer regularly running the PFN jobs, it also doesn't hurt to keep these here, right?

Reply:
I though they were done with, but I'll revive them!